pkg/dashboard/test/server_test.go

```diff
@@ -2370,13 +2373,15 @@ func (suite *miscTestSuite) TestGetFrontendSpec() {
                         "kind": "http",
                         "name": "default-http",
                         "maxWorkers": 1,
-                        "workerAvailabilityTimeoutMilliseconds": 10000
+                        "workerAvailabilityTimeoutMilliseconds": 10000,
+                        "serviceType": "NodePort"
```

Comment:
```suggestion
                        "serviceType": "NodePort",
```

Reply:
This is the expected json returned in the http request. There's no `,` there.

Reply:
@liranbg I wish it to be python dict too :(